backend/app/models/operation.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+# operations table
+class Operation < ActiveRecord::Base
+
+  def self.pending_operations(operation_ids)
+    sql = "select id, operation_type_id from operations where id in ( #{operation_ids.join(',')} ) and status = 'pending'"
+    operations = Operation.find_by_sql sql
+  end
+
+  def self.distinct_operation_types(operation_ids, status)
+    sql = "select distinct operation_type_id from operations where id in ( #{operation_ids.join(',')} ) and status = ?"
+    distinct = Operation.find_by_sql [sql, status]
+  end
+
+  def self.operations_for_category_type_status(category, type, status)
```

Comment:
It has to go somewhere.... It is the call that builds returns the list of operations for the manager page when filtering by operation type.

As we continue to build the system it will be generalized to include further customizations, but those are the filters we have now.  Let's discuss.